src/NuGet.Core/NuGet.Packaging/Rules/AnalysisResources.resx

```diff
@@ -126,6 +126,24 @@
   <data name="DefaultSpecValueWarning" xml:space="preserve">
     <value>The value "{0}" for {1} is a sample value and should be removed. Replace it with an appropriate value or remove it and rebuild your package.</value>
   </data>
+  <data name="DependenciesGroupsForEachTFMBeginningToFiles" xml:space="preserve">
+    <value>- Add lib or ref assemblies for the</value>
+  </data>
+  <data name="DependenciesGroupsForEachTFMBeginningToNuspec" xml:space="preserve">
```

Comment:
I'm not sure if `dependencies` group should be localized. 

@NuGet/nuget-client thoughts? 

My reasoning is that's what the name of the nuspec element is.

Reply:
I'm a little confused by what you mean. Are you saying that the name of the tag shouldn't be localized or its contents? Because here the value doesn't have `dependencies` in it

Reply:
I meant  to comment on the line below.

Localizing dependencies when actually referring to the nuspec dependencies section is confusing.

Reply:
So what are you suggesting? just pull the tag from the actual nuspec?

Reply:
Don't localize it. 
Be it by not including it in this string or adding a non localization comment to it (see other related comments). 

Currently it will get localized